modules/module-info.php

```diff
@@ -430,7 +430,8 @@ function jetpack_contact_form_more_info() {
 	echo '</p>';
 
 	echo '<p>';
-	_e( 'Each contact form can easily be customized to fit your needs. When a user submits your contact form, the feedback will be filtered through <a href="http://akismet.com/">Akismet</a> (if it is active on your site) to make sure it’s not spam. Any legitimate feedback will then be emailed to you, and added to your feedback management area.', 'jetpack' );
+    $plugin_url = admin_url( 'plugin-install.php?tab=search&s=akismet');
+    printf( __( 'Each contact form can easily be customized to fit your needs. When a user submits your contact form, the feedback will be filtered through <a href="http://akismet.com/">Akismet</a> (if it is <a href="%s">active on your site</a>) to make sure it’s not spam. Any legitimate feedback will then be emailed to you, and added to your feedback management area.', 'jetpack' ), $plugin_url);
```

Comment:
3 minor nitpicks:
- I'd place each `printf` parameter on a separate line, it makes things more readable.
- Why not including `admin_url( 'plugin-install.php?tab=search&s=akismet')` directly in `printf`, instead of creating that variable before?
- You'll want to add a space before `printf`'s closing bracket.